enterprise/views.py

```diff
@@ -234,6 +244,8 @@ def get_course_specific_consent(self, request, course_id):
 
         platform_name = configuration_helpers.get_value("PLATFORM_NAME", settings.PLATFORM_NAME)
         context_data = self.get_default_context(customer, platform_name)
+        global_context_data = get_global_context(request)
+        context_data.update(global_context_data)
```

Comment:
It would be good to pass the request to `get_default_context` and include the global values there.